File: 백준/Silver/2751. 수 정렬하기 2/수 정렬하기 2.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.util.Arrays;
import java.util.HashSet;
import java.util.Set;

public class Main {
    public static void main(String[] args) throws Exception {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();

        int n = Integer.parseInt(br.readLine());
        Set<Integer> set = new HashSet<>();
        for (int i = 0; i < n; i++) {
            int num = Integer.parseInt(br.readLine());
            set.add(num);
        }

        int arr[] = set.stream().mapToInt(Integer::intValue).toArray();
        Arrays.sort(arr);

        for (int i = 0; i < arr.length; i++) {
            sb.append(arr[i]).append("\n");
        }
        System.out.println(sb.toString());
    }
}

File: 백준/Silver/2751. 수 정렬하기 2/수 정렬하기 2.kt
import java.util.TreeSet

fun main() {
    val n = readLine()!!.toInt()
    val sb = StringBuilder()
    val set = TreeSet<Int>()

    for (i in 0 until n) {
        set.add(readLine()!!.toInt())
    }

    for (num in set) {
        sb.append(num).append("\n")
    }
    println(sb.toString())
}